### PR TITLE
Adding my website

### PR DESCRIPTION
**Pull request by martin69420**
the webring is located at the bottom of the main page https://martinpetkovski.com/

### CODE REVIEW

**Comment by neauoire** [comment]
Most things do not seem to work without javascript.

**Comment by martin69420** [comment]
> Most things do not seem to work without javascript.

This isn't true, pretty much everything on the website works without javascript, I tested it myself

The only things that don't work are the dropdown menu at the top, (the content inside is visible elsewhere on the page)

**Comment by neauoire** [comment]
Oh, let me try again, maybe I had some other issue that made it hard to navigate.
